avocado/plugins/run.py

```diff
@@ -101,10 +101,10 @@ def configure(self, parser):
         )
 
         settings.add_argparser_to_option(
-            namespace="run.test_runner",
+            namespace="run.suite_runner",
             parser=parser,
-            long_arg="--test-runner",
-            metavar="TEST_RUNNER",
+            long_arg="--suite-runner",
+            metavar="SUITE_RUNNER",
```

Comment:
IIUC, this change is breaking our API in the sense that users might need some time to get used to it. So it looks like we need a deprecation warning + keep supporting this for a while.